framework/src/modules/network/network.js

```diff
@@ -111,19 +111,18 @@ module.exports = class Network {
 		);
 
 		const seedPeers = await lookupPeersIPs(this.options.seedPeers, true);
-		const blacklistedPeers = this.options.blacklistedPeers
-			? this.options.blacklistedPeers.map(peer => ({
-					ipAddress: peer.ip,
-			  }))
-			: [];
+		const blacklistedPeers = this.options.blacklistedPeers || [];
```

Comment:
I suggest to rename this to `blacklistedIPs`.

Reply:
we will make this change in `3.0`